### PR TITLE
Removing hard-coded CONLL File header

### PR DESCRIPTION
**Pull request by lunayach**
(Solves #3) To accommodate different CONLL versions, I am using the `startsWith` method. Would be happy to discuss other possible solutions.

### CODE REVIEW

File: wildnlp/datasets/conll.py [hunk 1]
```diff
@@ -8,6 +8,7 @@ class CoNLL(Dataset):
     entity recognition. For details see:
     https://www.clips.uantwerpen.be/conll2003/ner/
     """
+    datasetFileHeader: str
```
Comment (adam-slucki) [inline wildnlp/datasets/conll.py]: Could you move this variable explicitly to the constructor and make it the private member?

```suggestion
    def __init__(self):
        self._dataset_file_header: str
        super().__init__()
```

File: wildnlp/datasets/conll.py [hunk 2]
```diff
@@ -32,7 +33,8 @@ def load(self, path):
         """
         with open(path, "r") as f:
             for line in f.readlines():
-                if line == "-DOCSTART- -X- O O\n":
+                if str(line).startswith('-DOCSTART-'):
+                    self.datasetFileHeader = str(line)
```
Comment (adam-slucki) [inline wildnlp/datasets/conll.py]:
```suggestion
                    self._dataset_file_header = str(line)
```

File: wildnlp/datasets/conll.py [hunk 3]
```diff
@@ -121,7 +123,7 @@ def save(self, data, path):
         with open(path, 'w+') as f:
 
             lines = list()
-            lines.append('-DOCSTART- -X- O O\n')
+            lines.append(self.datasetFileHeader)
```
Comment (adam-slucki) [inline wildnlp/datasets/conll.py]:
```suggestion
            lines.append(self._dataset_file_header)
```

**Comment by lunayach** [comment]
Hi @adam-slucki, Thanks for the reviews! Pushed the changes.

**Pull request merged**